src/triagebot/transfer.md

```diff
@@ -0,0 +1,32 @@
+# Issue Transfer
+
+The `transfer` command allows you to transfer a GitHub issue from one repository to another.
+
+## Usage
+
+To transfer an issue to another repository, enter a comment with the form:
+
+`@rustbot transfer` *repository-name*
+
+It is recommended to also include a comment explaining why you are transferring. For example:
+
+```
+Transferring to rust-lang/cargo since this is an issue with how cargo
+implements diagnostic reports.
+
+@rustbot transfer cargo
+```
+
+**IMPORTANT: There will be no visual indication that the issue is being transferred.** Due to GitHub API limitations, you will not see any activity. **You must reload the page** to view the issue in its new location. It may take a few moments for GitHub to transfer all the data.
+
+**WARNING:** Transferring is a partially destructive command. For example, labels and milestones that don't exist in the target repository will be removed.
+
+The transfer command is limited to team members of the rust-lang org, and transfers can only happen to repositories in the rust-lang org.
+
+## Configuration
+
+There is no configuration for this command. It is enabled for all rust-lang repositories.
```

Comment:
So why do we need https://github.com/rust-lang/rust/pull/124725? Do we not need it?

Reply:
We do seem to need it (https://github.com/rust-lang/rust/issues/124619#issuecomment-2094258528) since all(?) triagebot commands are “gated behind the existence of a config.”: https://github.com/rust-lang/triagebot/blob/da0d5acdf5412bf246b79aa7b3c9c3168ddb640a/src/handlers.rs#L253-L264

Reply:
Sorry, brainfart. I have fixed this.